client/components/planning-editor-standalone/field-definitions/place-field.ts

```diff
@@ -0,0 +1,38 @@
+import {IDropdownConfigVocabulary, IAuthoringFieldV2, IVocabularyItem} from 'superdesk-api';
+import {superdeskApi} from '../../../superdeskApi';
+import {IFieldGetter} from '.';
+
+export const getPlaceField: IFieldGetter = () => ({
```

Comment:
`IFieldGetter` doesn't server any purpose here compared to
`const getPlaceField = (): IFieldDefinition => {...}`